internal/httphandler/ledger_request_handler.go

```diff
@@ -62,6 +64,9 @@ func NewLedgerRequestHandler(db bcdb.DB, logger *logger.SugarLogger) http.Handle
 	handler.router.HandleFunc(constants.GetDataProofPrefix+"/{dbname}", handler.invalidDataProof).Methods(http.MethodGet)
 	// HTTP GET "/ledger/proof/data/{blockId}/{dbname}/{key}" with invalid query params
 	handler.router.HandleFunc(constants.GetDataProofPrefix+"/{dbname}/{key}", handler.invalidDataProof).Methods(http.MethodGet)
+	// HTTP GET "/ledger/tx/content/{blockId}?idx={idx}" with invalid query params
+	handler.router.HandleFunc(constants.GetTxContentPrefix, handler.invalidTxContent).Methods(http.MethodGet)
+	handler.router.HandleFunc(constants.GetTxContent, handler.invalidTxContent).Methods(http.MethodGet)
```

Comment:
Removing this will return a page not found (404) instead of a bad request, right?

Reply:
correct